src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -131,20 +143,31 @@ def generate_MCP_1_default_config(self, filename):
           open(os.path.join('testcases', 'testdata', 'cbsd_4.json')))
         sas_th_cbsd_5 = json.load(
           open(os.path.join('testcases', 'testdata', 'cbsd_5.json')))
-        sas_th_0_cbsd_iter_0 = [sas_th_cbsd_0]
-        sas_th_0_cbsd_iter_1 = [sas_th_cbsd_1]
-        sas_th_0_cbsd_iter_2 = [sas_th_cbsd_2]
-        sas_th_1_cbsd_iter_0 = [sas_th_cbsd_3]
-        sas_th_1_cbsd_iter_1 = [sas_th_cbsd_4]
-        sas_th_1_cbsd_iter_2 = [sas_th_cbsd_5]
-    
+        
+        sas_th_0_iteration_0_data = {'cbsdRecords':[sas_th_cbsd_0],
+                                      'escRecords': [esc_record_1],
+                                      'ppaRecords':[ppa_record_1] }
+        
+        sas_th_0_iteration_1_data = {'cbsdRecords':[sas_th_cbsd_1],
+                                      'escRecords': [esc_record_1],
+                                      'ppaRecords':[ppa_record_1] }
+        
+        sas_th_1_iteration_0_data = {'cbsdRecords':[sas_th_cbsd_0],
+                                      'escRecords': [esc_record_1],
+                                      'ppaRecords':[ppa_record_1] }
+        
+        sas_th_1_iteration_1_data = {'cbsdRecords':[sas_th_cbsd_1],
+                                      'escRecords': [esc_record_1],
+                                      'ppaRecords':[ppa_record_1] }
+ 
+          
         sas_harness_0 = {
                     'name': 'SAS-TH-1',
                     'url': 'localhost:9001',
                     'certFile': 'server_a.cert',
                     'keyFile': 'server_a.key',
                     'certificateHash': 'will be provided in future',
-                    'cbsdRecord': [sas_th_0_cbsd_iter_0, sas_th_0_cbsd_iter_1, sas_th_0_cbsd_iter_2],
+                    'sasThIterationData': [sas_th_0_iteration_0_data,sas_th_0_iteration_1_data],
```

Comment:
I'm concerned about the per-iteration data being far from the other per-iteration data, and I don't see a clear benefit to having it here.

Reply:
This   part of the config information is used by SAS Test Harness, so we want to   keep all the SAS TH required data in one structure at the same time want to   differentiate data records per iteration based.

Reply:
I think it will be much easier to read and debug if we keep it with the other per-iteration data.

Reply:
updated,   now placed along with other iteration data.